graylog2-server/src/main/java/org/graylog2/security/ldap/LdapConnector.java

```diff
@@ -326,7 +326,7 @@ private String normalizedDn(String dn) {
             try {
                 return new Dn(dn).getNormName();
             } catch (LdapInvalidDnException e) {
-                LOG.warn("Invalid DN", e);
+                LOG.debug("Invalid DN", e);
```

Comment:
Maybe the method is not named correctly. We want to use this to:

1. normalize DN strings if the passed value is a DN
1. if it's not a DN we do not have to normalize it and just return the original

We need this because in the following case the member attribute might be a DN string or a regular string like just the username.

```java
LOG.trace("DN {} == {} member?", dn, member.getString());
if (dn.equalsIgnoreCase(normalizedDn(member.getString()))) {
    groups.add(groupId);
} else {
```

We do not really care if it's a DN or not. The important part is that if the value is a DN it should be normalized.